create_seed_checkpoint.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/bash
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+set -ex
+
+# libUV is a scalable backend for TCPStore which is used in processGroup
+# rendezvous. This is the recommended backend for distributed training.
+export USE_LIBUV=1
+TRAINER_DIR=${1:-/home/$USER/local/torchtitan}
+
+# use envs as local overrides for convenience
+# e.g.
+# LOG_RANK=0,1 NGPU=4 ./run_llama_train.sh
+
+NGPU=1
+LOG_RANK=0
+
+
+CONFIG_FILE=${CONFIG_FILE:-"./train_configs/debug_model.toml"}
+
+seed_checkpoint="--checkpoint.enable_checkpoint --checkpoint.create_seed_checkpoint"
+overrides=""
+if [ $# -ne 0 ]; then
+    overrides="$*"
+fi
+
+torchrun --nproc_per_node=${NGPU} --rdzv_backend c10d --rdzv_endpoint="localhost:0" \
+--local-ranks-filter ${LOG_RANK} --role rank --tee 3 \
+train.py --job.config_file ${CONFIG_FILE} $seed_checkpoint $overrides
```

Comment:
If `NGPU` is always 1 for creating seed checkpoint, shall we just launch the script by:
`python train.py`
?
Not sure if it would work directly though.